code/espurna/system.ino

```diff
@@ -243,7 +243,7 @@ void _systemSetupSpecificHardware() {
 
     // These devices use the hardware UART
     // to communicate to secondary microcontrollers
-    #if (RF_SUPPORT && !RFB_DIRECT) || (RELAY_PROVIDER == RELAY_PROVIDER_DUAL) || (RELAY_PROVIDER == RELAY_PROVIDER_STM)
+    #if (RF_SUPPORT && !RFB_DIRECT) || (RELAY_PROVIDER == RELAY_PROVIDER_DUAL) || (RELAY_PROVIDER == RELAY_PROVIDER_STM) || (RELAY_PROVIDER == RELAY_PROVIDER_HW655)
```

Comment:
we need Serial communication initialized (control of HW-655 relays is done through serial port)

Reply:
Yes, this should probably be a runtime flag, but we never got around managing debugging like that bc that depends on serial begin too (and reverse-depends on this condition)